cves/kernel/CVE-2013-0349.yml

```diff
@@ -305,9 +336,13 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: |
+    There were statements made about the vulenrability such as the commit
+    message for fixing it explaining teh vulnerability and the fix applied,
+    or the request for a CVE or bugzilla report, but no dsicussions were sparked
```

Comment:
If the comments had some level of back and forth it might still be worth considering it a discussion.

Reply:
There was no level of back and forth with any form of meaning, hence why I claimed they were statements and potential acknowledgments. However, no level of discussion is formed if the discussion is defined by two or more parties meaningfully communicating back and forth.